### ISSUE TITLE
Language for Official US Government Banner

### ISSUE THREAD

**Issue by bengm** [USAJOBS/design-system#118]
There is a small language/content issue in the banner when viewing the design system. 
The URL (for the design system, not the core app) is "http://usajobs.github.io/design-system/" and is a ".io" domain. Perhaps the banner should not be on the design system? Or perhaps the paragraph should be updated to explain it away? Alternatively, could use a custom subdomain to point to the github pages site, like "designsystem.usajobs.gov"?

![image](https://user-images.githubusercontent.com/819222/35352548-52049486-0112-11e8-8463-8d22ca8acadd.png)

**Comment by humancompanion** [comment]
Yes, @bengm, that's a fair point. We are working towards a sub-domain for the design system. I don't want to remove the banner because it is an official website, or at least not an imposter.

**Comment by humancompanion** [comment]
@kikaider05 - Would it be possible to get a sub-domain of design-system.usajobs.gov setup? If so, I can deploy through Federalist.

**Comment by humancompanion** [comment]
We are working the switch to GSA's Federalist through CIO approvals. It will take some time but that would unblock us to use a sub-domain and still deploy the design system directly from our git repo.
